video/out/wayland_common.c

```diff
@@ -3532,6 +3532,7 @@ static void set_color_representation(struct vo_wayland_state *wl)
     int coefficients = wl->coefficients_map[repr.sys];
     int range = repr.levels == PL_COLOR_LEVELS_FULL ? wl->range_map[repr.sys] :
                                 wl->range_map[repr.sys + PL_COLOR_SYSTEM_COUNT];
+    bool is_420_subsampled = mp_imgfmt_is_420_subsampled(wl->target_params.hw_subfmt);
```

Comment:
Is `hw_subfmt` always available? It should fallback to `imgfmt` if not.

Reply:
I couldn’t figure out how to get it to not use vaapi, even `--hwdec=no` says “[autoconvert] HW-uploading to vaapi”, but here is an attempt at doing that fallback.